app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/ItemLauncher.java

```diff
@@ -143,7 +143,7 @@ public static void launch(final BaseRowItem rowItem, ItemRowAdapter adapter, int
                         MediaManager mediaManager = KoinJavaComponent.<MediaManager>get(MediaManager.class);
 
                         // if the song currently playing is selected (and is the exact item - this only happens in the nowPlayingRow), open AudioNowPlayingActivity
-                        if (mediaManager.hasAudioQueueItems() && rowItem.getBaseItem() == mediaManager.getCurrentAudioItem()) {
+                        if (mediaManager.hasAudioQueueItems() && rowItem instanceof AudioQueueItem && rowItem.getBaseItem().getId().equals(mediaManager.getCurrentAudioItem().getId())) {
```

Comment:
Why not
```suggestion
                        if (mediaManager.hasAudioQueueItems() && rowItem instanceof AudioQueueItem && rowItem.getBaseItem().getId() ==mediaManager.getCurrentAudioItem().getId()) {
```

?

Reply:
That's what I did originally but the android studio linter said I should use .equals()